R/carehomes.R

```diff
@@ -448,6 +460,20 @@ carehomes_initial <- function(info, n_particles, pars) {
 }
 
 
+##' Carehomes vaccination parameters.
+##'
+##' @title Carehomes vaccination parameters
+##'
+##' @return A list of parameter values
+##'
+##' @export
+carehomes_parameters_vaccination <- function(rel_susc = 1) {
+  list(N_vacc_classes = length(rel_susc),
```

Comment:
Do you make any assumptions about the patterns of `rel_susc`? Must it be monotonic if it has a length of greater than 1 for example?

Reply:
Typically we will have 
rel_susc <- c(1, vacc, waning_1, waning_2, ...) 
where vacc < waning_1 < waning_2 < ... < 1
I will write this in a function which will check the format is that expected and give a warning if not I think, if that seems sensible?

Reply:
yes, sounds good

Reply:
Ah, actually no - don't give a warning, please throw an error

Reply:
@richfitz has now recoded these checks